In file included from ../../Clementine-1.2.2/src/core/mergedproxymodel.cpp:18:0:
../../Clementine-1.2.2/src/core/mergedproxymodel.h:23:12: error: 'std::placeholders' has not been declared
 using std::placeholders::_1;
../../Clementine-1.2.2/src/core/mergedproxymodel.h:24:12: error: 'std::placeholders' has not been declared
 using std::placeholders::_2;


--- a/src/core/mergedproxymodel.h
+++ b/src/core/mergedproxymodel.h
@@ -19,6 +19,7 @@
 #define MERGEDPROXYMODEL_H
 
 #include <QAbstractProxyModel>
+#include <functional>
 
 using std::placeholders::_1;
 using std::placeholders::_2;
